internal/appsec/config.go

```diff
@@ -109,6 +121,25 @@ func readRateLimitConfig() (rate uint) {
 	return uint(parsed)
 }
 
+func readObfuscatorConfig() *ObfuscatorConfig {
```

Comment:
Go tip: it is okay returning small and temporary structs by copy
```suggestion
func readObfuscatorConfig() ObfuscatorConfig {
```